tests/e2e/tour/tour.spec.js

```diff
@@ -103,6 +108,77 @@ test.describe('When loading the page with dontShowTour preference set to true',
   })
 })
 
+test.describe('When logged in', () => {
+  test.beforeEach(async ({ page, context }) => {
+    await setupTests({
+      page,
+      context,
+      dontShowTour: true
+    })
+
+    await login(context)
+
+    await page.route(/collections.json/, async (route) => {
+      await route.fulfill({
+        json: singleCollection.body,
+        headers: singleCollection.headers
+      })
+    })
+
+    await page.goto('/search')
+  })
+
+  test('should see the tour slide for non-logged in users', async ({ page }) => {
+    // Start the tour by clicking the "Start Tour" button
+    page.getByRole('button', { name: 'Start tour' }).click()
+
+    // Start Tour View: Welcome to Earthdata Search
+    await expect(page.locator('.search-tour__welcome')).toContainText('Welcome to Earthdata Search!')
```

Comment:
Can we change the `page.locator` calls in this file to use `page.getByRole('alertdialog', { value: /Some text/ })` like in the tests for hiding the tour? 

This is just to better match our testing thought process of testing things the user can see, and not code specific things. Similar to switch to React Testing Library and looking for roles instead of test ids for things